test/engine-tap/gh-6780-before-replace-trigger-assertion.test.lua

```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/env tarantool
```

Comment:
Please rewrite the test with luatest. TAP tests are discouraged.

Reply:
Since I already wrote a test with TAP, there is no need in rewriting it. I will write new tests with luatest but let's leave this one as it is.